### ISSUE TITLE
loss sudden increase from 0.2 to 40, and preview goes black

### ISSUE THREAD

**Issue by vrvc014pssi** [iperov/DeepFaceLab#198]
## Expected behavior

Traning goes fine.

## Actual behavior

![1](https://user-images.githubusercontent.com/48871944/54873935-35ca1e00-4e1c-11e9-9cdd-106ff63c63a3.jpg)

Before 61410, training goes fine without any problem.
But after that, AI preview goes black, source image preview still showing through.
There are 5 columns in preview window, column 1,3 are showing, column 2, 4, 5 are completely black.

[02:43:23][#061370][4251ms] src_loss:0.286 dst_loss:0.137
[02:44:03][#061380][4196ms] src_loss:0.211 dst_loss:0.101
[02:44:43][#061390][3925ms] src_loss:0.221 dst_loss:0.176
[02:45:33][#061400][4071ms] src_loss:0.228 dst_loss:0.171
[02:46:27][#061410][4233ms] src_loss:18.271 dst_loss:40.176
[02:47:02][#061420][4122ms] src_loss:19.5572 dst_loss:40.248
[02:47:51][#061430][4235ms] src_loss:17.217 dst_loss:39.314
[02:48:37][#061440][4295ms] src_loss:18.575 dst_loss:40.589

![66](https://user-images.githubusercontent.com/48871944/54874013-de2cb200-4e1d-11e9-91f6-cc5f8b481261.jpg)

And this grapth become completely black after 61410

## Steps to reproduce

Not yet

## Other relevant information
- **Command lined used (if not specified in steps to reproduce)**: 
`$DFL_PYTHON "$DFL_SRC/main.py" train \
    --training-data-src-dir "$DFL_WORKSPACE/data_src/aligned" \
    --training-data-dst-dir "$DFL_WORKSPACE/data_dst/aligned" \
    --model-dir "$DFL_WORKSPACE/model" \
    --model SAE`

- Operating system and version:Ubuntu 16.04
- Python version:3.6.8

<pre>
Loading model...
Press enter in 2 seconds to override model settings.Using TensorFlow backend.
Loading: 100%|#############################| 4093/4093 [00:06<00:00, 626.97it/s]
Loading: 100%|###########################| 10115/10115 [00:17<00:00, 572.99it/s]
===== Model summary =====
== Model name: SAE
==
== Current iteration: 55633
==
== Model options:
== |== batch_size : 16
== |== sort_by_yaw : False
== |== random_flip : False
== |== resolution : 256
== |== face_type : f
== |== learn_mask : True
== |== optimizer_mode : 3
== |== archi : liae
== |== ae_dims : 512
== |== ed_ch_dims : 50
== |== ca_weights : True
== |== lighter_encoder : False
== |== multiscale_decoder : True
== |== pixel_loss : True
== |== face_style_power : 1.0
== |== bg_style_power : 1.0
== |== write_preview_history : True
== Running on:
== |== [0 : Tesla P100-PCIE-16GB]
=========================

</pre>

**Comment by iperov** [comment]
retrain with last commit

**Comment by iperov** [comment]
and train without CA weights

**Comment by vrvc014pssi** [comment]
<pre>Running trainer.

Loading model...
Press enter in 2 seconds to override model settings.Using TensorFlow backend.
Loading: 100%|#############################| 4093/4093 [00:05<00:00, 726.70it/s]
Loading: 100%|###########################| 10115/10115 [00:15<00:00, 649.81it/s]
===== Model summary =====
== Model name: SAE
==
== Current iteration: 57864
==
== Model options:
== |== write_preview_history : True
== |== batch_size : 16
== |== sort_by_yaw : False
== |== random_flip : False
== |== resolution : 256
== |== face_type : f
== |== learn_mask : True
== |== optimizer_mode : 3
== |== archi : liae
== |== ae_dims : 512
== |== ed_ch_dims : 64
== |== ca_weights : False
== |== lighter_encoder : False
== |== multiscale_decoder : True
== |== pixel_loss : True
== |== face_style_power : 0.0
== |== bg_style_power : 0.0
== Running on:
== |== [0 : Tesla V100-PCIE-16GB]
=========================
Starting. Press "Enter" to stop training and save model.
[05:00:29][#061716][5436ms] src_loss:0.056 dst_loss:0.079
[05:15:33][#061883][5404ms] src_loss:0.073 dst_loss:0.062
[05:30:37][#062051][5243ms] src_loss:0.074 dst_loss:0.073
[05:45:40][#062219][4925ms] src_loss:0.065 dst_loss:0.060
[06:00:41][#062387][5457ms] src_loss:0.095 dst_loss:0.076
[06:15:44][#062555][5381ms] src_loss:0.074 dst_loss:0.055
[06:30:47][#062722][4897ms] src_loss:0.075 dst_loss:0.058
[06:45:52][#062891][5200ms] src_loss:0.065 dst_loss:0.068
[07:00:53][#063058][5318ms] src_loss:0.058 dst_loss:0.055
[07:15:56][#063226][5140ms] src_loss:0.074 dst_loss:0.076
[07:30:57][#063394][5533ms] src_loss:0.071 dst_loss:0.036
[07:45:59][#063563][4808ms] src_loss:0.075 dst_loss:0.054
[08:01:03][#063731][5501ms] src_loss:0.081 dst_loss:0.068
[08:16:08][#063898][5347ms] src_loss:0.080 dst_loss:0.065
[08:31:11][#064065][5321ms] src_loss:0.073 dst_loss:0.049
[08:46:15][#064233][5191ms] src_loss:0.077 dst_loss:0.045
[09:01:19][#064401][5176ms] src_loss:18.048 dst_loss:29.288
[09:16:19][#064568][4987ms] src_loss:19.736 dst_loss:27.891
[09:31:20][#064735][5521ms] src_loss:18.324 dst_loss:27.965
[09:46:21][#064903][5219ms] src_loss:19.172 dst_loss:24.991
[10:01:21][#065071][4984ms] src_loss:19.276 dst_loss:27.339
[10:16:25][#065239][5085ms] src_loss:18.450 dst_loss:26.516
[10:31:28][#065408][5244ms] src_loss:18.088 dst_loss:29.540
[10:46:31][#065576][5137ms] src_loss:18.265 dst_loss:27.306
[11:01:32][#065744][5254ms] src_loss:17.107 dst_loss:27.127
[11:16:35][#065912][5001ms] src_loss:17.800 dst_loss:26.808
[11:31:35][#066080][5275ms] src_loss:19.346 dst_loss:27.911
[11:46:36][#066248][4629ms] src_loss:17.571 dst_loss:24.329
[12:01:41][#066416][5408ms] src_loss:18.486 dst_loss:27.478
[12:03:54][#066440][5267ms] src_loss:18.672 dst_loss:29.480</pre>

Disable CA weights, retrain from scratch.
Still same.

**Comment by iperov** [comment]
no solution.
